scripts/delete_cluster.sh

```diff
@@ -6,18 +6,22 @@ CLUSTER_NAME="$1"
 REGION="$2"
 export AWS_DEFAULT_REGION=${REGION}
 echo "Inputs CLUSTER_NAME  : ${CLUSTER_NAME} , REGION: ${REGION} "
-BIN_DIR=$(cat .bin_dir)
-cat .bin_dir
 export ROSA_TOKEN="$3"
+# BIN_DIR=$(cat .bin_dir)
```

Comment:
Are these commented sections still needed?  If not, maybe remove them to get rid of unnecessary code?

Reply:
Removed comments